api/illumio/cloud/k8sclustersync/v1/k8s_info.proto

```diff
@@ -24,6 +24,27 @@ message KubernetesObjectData {
   oneof kind_specific {
     KubernetesPodData pod = 100;
     KubernetesNodeData node = 101;
+    KubernetesServiceData service = 102;
+  }
+}
+
+message KubernetesServiceData {
```

Comment:
Add the missing fields from your `ServiceAttributes` struct.
Most of them should be `optional`.